src/amr/tagging/default_hybrid_tagger_strategy.h

```diff
@@ -43,17 +43,11 @@ void DefaultHybridTaggerStrategy<HybridModel>::tag(HybridModel& model,
     {
         for (auto iCell = 0u, ix = start; iCell <= endCell; ++ix, ++iCell)
         {
-            auto Byavgm2 = 0.2 * (By(ix - 4) + By(ix - 3) + By(ix - 2) + By(ix - 1) + By(ix));
-            auto Byavgm1 = 0.2 * (By(ix - 3) + By(ix - 2) + By(ix - 1) + By(ix) + By(ix + 1));
             auto Byavg   = 0.2 * (By(ix - 2) + By(ix - 1) + By(ix) + By(ix + 1) + By(ix + 2));
             auto Byavgp1 = 0.2 * (By(ix - 1) + By(ix) + By(ix + 1) + By(ix + 2) + By(ix + 3));

```

Comment:
is the (ix + 3) not a bit suspect for interp1? (ie. two ghosts)

Reply:
looks like https://hephaistos.lpp.polytechnique.fr/teamcity/buildConfiguration/Phare_Phare_BuildGithubPrClang/71213?buildTab=log&focusLine=4776&linesState=1063

Reply:
you are right. We should probably not take 5 points when on patch borders

Reply:
Next commit should address that issue